### PR TITLE
Prepare for the WG release

### PR DESCRIPTION
**Pull request by xstefank**
Signed-off-by: xstefank <xstefank122@gmail.com>

### CODE REVIEW

**Review by ederks85 — APPROVED** [review]

**Review by Emily-Jiang — CHANGES_REQUESTED** [review]
You need to copy the file license-efsl.adoc

File: spec/src/main/asciidoc/license-efsl.adoc [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+....
+Specification: {doctitle}
+Version: {revnumber}
+Status: {revremark}
+Release: {revdate}
+....
+
+== Copyright
+
+Copyright (c) 2016, 2020 Eclipse Foundation.
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/license-efsl.adoc:11]: You should use inceptionYear and currentYear here. See [here](https://github.com/eclipse/microprofile-fault-tolerance/blob/master/spec/src/main/asciidoc/license-efsl.adoc). It was further improved after Health was released.

Comment (Emily-Jiang) [reply spec/src/main/asciidoc/license-efsl.adoc:11]: You need to copy the inceptionYear and currentYear declaration in pom.xml

Comment (xstefank) [reply spec/src/main/asciidoc/license-efsl.adoc]: thanks, added the suggestion.

**Review by radcortez — APPROVED** [review]

**Review by Emily-Jiang — APPROVED** [review]

**Pull request merged**